deploy/backplane/20-cee.SubjectPermission.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-cee
+  namespace: openshift-rbac-permissions
+spec:
```

Comment:
Rather we're explicit on what the allow / deny regex are even if it defaults to something.  Is the intent of this to have `namespacesAllowedRegex: ".*"`?

Reply:
yes the intent was to allow for all namespaces and I was under the impression that adding a clusterPermission by default applies to all namespaces. The confusing bit was the difference between clusterPermission and permission in the subjectPermission.yml file.

so does this look better instead?

```
spec:
  permissions:
  - allowFirst: true
    clusterRoleName: backplane-cee
    namespacesAllowedRegex: .*
  subjectKind: Group
  subjectName: system:serviceaccounts:openshift-backplane-cee
```